resources/assets/less/bem/forum-topic-nav.less

```diff
@@ -25,6 +25,7 @@
     position: absolute;
     bottom: 0;
     left: 0;
+    overflow-x: auto;
```

Comment:
`forum-topic-nav` block can use `u-fancy-scrollbar` class so the scrollbar isn't oversized on desktop browsers.

Reply:
Probably the scrollbar will not show in desktop because most likely it will never overflow, but I'll add it.